app/javascript/components/request-workflow-status/data.js

```diff
@@ -2,6 +2,7 @@
 export const workflowStateTypes = {
   success: { text: 'success', tagType: 'green' },
   error: { text: 'error', tagType: 'red' },
+  failure: { text: 'failure', tagType: 'red' },
```

Comment:
This is unexpected.  Did something change?

Reply:
I noticed an error on the service request page where it was returning failure for the status and since it was missing was causing the component to break on the page. The workflow tasks also return failure. I don't think anything changed, I think we were just missing the failure state for this component.

Reply:
Yeah I think the workflow status column should only ever have `["pending", "running", "failure", "success"]` so not sure why we have `error` and `failed` here, maybe in a follow-up we see if we can remove those

Reply:
error might have been from before you were capturing errors and turning those into failures? Like if the code raised an exception all the way up to the top?